dali_tf_plugin/build_in_custom_op_docker.sh

```diff
@@ -7,6 +7,11 @@ PYTHON_DIST_PACKAGES=( $(python -c "import site; print(site.getsitepackages()[0]
 DALI_TOPDIR="${PYTHON_DIST_PACKAGES}/nvidia/dali"
 
 CUDA_VERSION=$(cat /usr/local/cuda/version.txt | head -1 | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1\2/')
+# when building for any version >= 11.0 use CUDA compatibility mode and claim it is a CUDA 110 package
```

Comment:
```
CUDA_VERSION="${CUDA_VERSION%?}0"
```
or 
```
CUDA_MAJOR_VERSION=$(cat /usr/local/cuda/version.txt | head -1 | sed 's/.*Version \([0-9]\+\)\.[0-9]\+.*/\1/')
CUDA_VERSION="${CUDA_MAJOR_VERSION}0"
```

Reply:
Not really. What if we want to build plugin for Jetson with 10.2? What if the user want to build with just 10.1 on his machine?

Reply:
Anyway, I have simplified it to `CUDA_VERSION="${CUDA_VERSION%?}0"` but I'm keeping the if.